constellations/gemini-admin/src/utils/textInput.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+require("./textInput.sass");
+import React from "react";
+import Checkbox from "../../assets/checkmark_green.svg";
+
+export class TextInput extends React.Component {
+    render = () => {
+        const description = this.props.description;
+        const required = this.props.required;
+        const value = this.props.value;
+
+        var format = <h4 className="hidden"></h4>;
+        if (!!description) {
+            let ending = required ? (
+                <span className="red">{" (required)"}</span>
```

Comment:
One more functionality of this `required` part. If the input is satisfied (error message goes away and checkmark appears), turn this red into the normal font color.
The idea is that if the user did everything correctly, they should see no red on the page.